src/index.html

```diff
@@ -11,7 +11,214 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" class="form">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div>
```

Comment:
Be consistant and add class to all form fields wrappers. You can then specify the margins using pseudo selectors in css

Reply:
All tips done I hope now its good.